### ISSUE TITLE
NonText Background and CursorLine issue

### ISSUE THREAD

**Issue by noenthu** [ntk148v/vim-horizon#5]
When enabling cursorline and having listchars with eol etc values the background for the NonText char does not align with other themes

![image](https://user-images.githubusercontent.com/989601/84069708-92b5e380-a990-11ea-8a3a-dfe3df53724e.png)

Here is what it looks like on neovim with another theme (similar behavior with gruvbox)
![image](https://user-images.githubusercontent.com/989601/84069900-df99ba00-a990-11ea-8854-3a12d796fc0f.png)

**Comment by ntk148v** [comment]
Hi @noenthu sorry for the late response,

Could you show me your vimrc and the sample file to reproduce this issue? Does you pull the newest vim-horizon source?

**Comment by noenthu** [comment]
@ntk148v Yes, this is using the latest version of the plugin (from vim-plug)

Seems the exact issue is with neovim.



On vim, is is a slightly different issue. The listchars dont appear on the current row

![image](https://user-images.githubusercontent.com/989601/104198831-534eea00-53ec-11eb-97eb-4d3f4d66a73c.png)

[vimrc.txt](https://github.com/ntk148v/vim-horizon/files/5796491/vimrc.txt)

**Comment by chaitanyabsprip** [comment]
it is not showing probably because the cursor line color is same as the listchar color
